### DISCUSSION TITLE
Link to codegen.go in mdatagen README does not exist anymore

### DISCUSSION THREAD

**Discussion by dgoscn** [open-telemetry/opentelemetry-collector-contrib#16238] [Q&A]
Hello guys. I was using the document for Metadata Generator, but, the link that points to the `codegen.go` from example, doesn't exists anymore. 

Someone knows if there are another path to resolve the same problem?

The link that not work is:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/codegen.go

That is descripted here:
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/mdatagen#build

Thank you

**Comment by jpkrohling** [comment]
Hi @dgoscn, I think we talked via Slack about it. Would you be able to send a PR to fix it?

**Reply by dgoscn** [reply]
Sure @jpkrohling

**Reply by dgoscn** [reply]
@jpkrohling I've opened this PR containing some fresh info 
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16770
